be/src/exec/vectorized/chunks_sorter.cpp

```diff
@@ -73,34 +73,34 @@ Status DataSegment::get_filter_array(std::vector<DataSegment>& data_segments, si
             for (size_t j = 0; j < rows; ++j) {
                 if (compare_results_array[i][j] < 0) {
                     filter_array[i][j] = DataSegment::BEFORE_LAST_RESULT;
```

Comment:
What's the meaning of DataSegment::BEFORE_LAST_RESULT and DataSegment::IN_LAST_RESULT;?

Reply:
We calling the sorted part as `Segment`. When new chunks coming, we will use the `Segment` to filter the chunks into tree parts
1. part 1,  `value < smallest(Segment)`, which is called `BEFORE_LAST_RESULT `
2. part2, `smallest(Segment) <= value <=largest(Segment)`, which is called `IN_LAST_RESULT `
3. part3, `value > largest(Segment)`